### PR TITLE
miniweb: minibuilder

### PR DESCRIPTION
**Pull request by aherna**

### CODE REVIEW

File: misc/web/grapheditor/index.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
+<!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=5,IE=9" ><![endif]-->
```
Comment (csymonds) [inline misc/web/grapheditor/index.html:1]: Sandia copyright statement

**Review by csymonds — CHANGES_REQUESTED** [review]
Please add copyright as needed

File: misc/web/grapheditor/index.html (hunk 1 above)
Comment (aherna) [reply misc/web/grapheditor/index.html:1]: Done @csymonds

**Review by csymonds — APPROVED** [review]
LGTM

**Pull request merged**